scripts/win_install.ps1

```diff
@@ -88,7 +87,9 @@ function Install-Dependencies {
     }
     Set-ItemProperty -Path 'Registry::HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Session Manager\Environment' -Name PATH -Value $newpath
     Refresh-Environment
-    
+
+    yarn global add serverless@2.29.0
```

Comment:
Installing `serverless` globally may not be necessary. This runs the risk that the serverless installed globally does not match the version installed locally per `package.json`

This can be a separate change though.

Reply:
Updating to use the `serverless` package that is locally installed and removing global `severless` installation.